integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -547,4 +547,4 @@ csi:
     - entry_timeout=36000
     - attr_timeout=36000
     - max_readahead=0
-  javaOptions: "-Dalluxio.user.metadata.cache.enabled=true "
+  javaOptions: "-Dalluxio.user.metadata.cache.enabled=true"
```

Comment:
nit: I don't have the full picture how this property is used, but if this does not break your code, I would recommend keeping the trailing whitespace. So later somewhere else if people append to this property they don't get an error.

Reply:
Thx, will fix it,.